### ISSUE TITLE
Tsx parser imports hoisted lodash version

### ISSUE THREAD

**Issue by DuncanWalter** [facebook/jscodeshift#418]
[This line](https://github.com/facebook/jscodeshift/blob/5910a4dcde431813c2e34a3a6f47e8800d275eb6/parser/tsx.js#L11) can cause transforms that have a second transitive dep on lodash to crash unexpectedly when lodash is unhoisted. I think this could be addressed a couple ways- either jscodeshift could express a dep on lodash or the tsx parser could remove the reference to lodash. I lean towards removing the lodash usage entirely (or at least not importing all of lodash to call one util one time).

**Comment by milesj** [comment]
I think I'm consistently hitting this.

```
Transformation error (Cannot find module 'lodash' Require stack: - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/parser/tsx.js - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/getParser.js - /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/Worker.js)
Error: Cannot find module 'lodash'
Require stack:
- /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/parser/tsx.js
- /Users/milesjohnson/.npm/_npx/19566/lib/node_modules/jscodeshift/src/getParser.js
- ```

**Comment by Daniel15** [comment]
Fixed in 0.13.0.
